packages/aws-cdk-lib/aws-opensearchservice/lib/domain.ts

```diff
@@ -511,6 +557,17 @@ export interface DomainProps {
    * @default - no custom domain endpoint will be configured
    */
   readonly customEndpoint?: CustomEndpointOptions;
+
+  /**
+   * Options for a domain's off-peak window, during which OpenSearch Service can perform mandatory
+   * configuration changes on the domain.
+   */
+  readonly offPeakWindowOptions?: OffPeakWindowOptions;
```

Comment:
specifically the default for this prop may be confusing. looks like the default should be that it is configured by default and cannot be turned off, but still, what will the default window be?

> Off-peak windows were introduced on February 16, 2023. All domains created before this date have the off-peak window disabled by default. You must manually enable and configure the off-peak window for these domains. All domains created after this date will have the off-peak window enabled by default. You can't disable the off-peak window for a domain after it's enabled.

https://docs.aws.amazon.com/opensearch-service/latest/developerguide/off-peak.html

Reply:
I defaulted it to `00:00 UTC`

> If you don't specify a custom window start time, it defaults to 00:00 UTC.